indexer/packages/postgres/src/db/migrations/migration_files/20230922141210_create_perpetual_positions_subaccount_perpetual_created_at_index.ts

```diff
@@ -0,0 +1,18 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  // eslint-disable-next-line @typescript-eslint/quotes
+  await knex.raw(`
+    CREATE INDEX CONCURRENTLY IF NOT EXISTS "perpetual_positions_subaccount_perpetual_created_at_index" ON "perpetual_positions" ("subaccountId", "perpetualId", "createdAtHeight");
```

Comment:
nit: Do you think it'd be possible to make this a partial index on the status of the position? E.g. we wouldn't touch `CLOSED` positions so there's no need to keep them in the index? Just want to ensure our indices don't grow too large.

Reply:
in both order fill/liquidation handler, we grab the latest perpetual position like so: 
    SELECT * INTO perpetual_position_record FROM perpetual_positions WHERE "subaccountId" = subaccount_uuid
                                                                       AND "perpetualId" = perpetual_market_record."id"
                                                                       ORDER BY "createdAtHeight" DESC;

Reply:
Would that work or would it break the logic?

Reply:
I don't think we are guaranteed that the last perpetual position is necessarily `OPEN` - so that would break the existing logic.